tracer/src/Datadog.Trace/OTelMetrics/MetricPoint.cs

```diff
@@ -33,6 +33,10 @@ internal class MetricPoint(string instrumentName, string meterName, InstrumentTy
 
     public Dictionary<string, object?> Tags { get; } = tags;
 
+    public string Unit { get; } = unit ?? string.Empty;
```

Comment:
nit: Can the inputs `unit` and `description` be null? It seems like you'll never hit the null-coalescing operator `??`